jumeg_causality/dirs_manage.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
@dongqunxi Hi Qunxi, (?) if this is some code copy pasted from Evernote, I think the formatting is messed up. Would you like to recheck?